config/vendor_api/draft.yml

```diff
@@ -103,3 +109,84 @@ components:
           description: Optional details about why the application was rejected for the given reason.
           example: You did not attend any of the arranged interviews.
           maxLength: 65535
+    HESAData:
+      type: object
+      description: |
+        Values to populate this candidate’s HESA Initial Teacher
+        Training data return.  Available once an offer has been accepted.
+      required:
+        - sex
+        - disability
+        - ethnicity
+      properties:
+        sex:
+          type: string
+          nullable: true
+          description: The candidate’s sex as a [2-digit HESA code for Sex](https://www.hesa.ac.uk/collection/c22053/e/sexid)
+          example: "10"
+          enum:
+          - "10"
+          - "11"
+          - "12"
```

Comment:
Should we include `96` (Information refused) if the candidate selected 'Prefer not to say' or should we continue to make the field nullable instead?

There's also `99` (‘Not available’: this code should be used in cases where it is not possible to supply the information such as a cyber-attack) which hopefully we'll never need?

Reply:
the reason some of these codes are not included in the spec is because it appears that we've not included them in our available codes – i can update the spec and then update the codes we have in our config